### ISSUE TITLE
Wallet not synchronizing accurately on MacBook Air

### ISSUE THREAD

**Issue by geofff2** [PirateNetwork/PirateWallet-Lite#77]
I have my wallet synchronizing on my Windows laptop properly and accurately and I have 4 deposits.

I have just installed PirateWallet-Lite on my new MacBook Air.  After synchronizing I find I only have 2 of my 4 deposits.  I have tried a couple of times with the same result.  

Thank you.

I have posted my latest information below.

**Comment by geofff2** [comment]
I have had no other result and nothing from this post to suggest a fix.

So I am downloading he Full Node wallet.  Cannot trust this one.

**Comment by geofff2** [comment]
Hi, 

I had funds in 3 addresses on the lite wallet on Windows which included 2 deposits to the same address and 2 deposits to different addresses.

I installed into my MacBook and found that only the 2 deposits in the same address are listed. I did a rescan on my Windows machine and I still had all of the deposits. But now my password did not work. So I deleted the lite wallet off the Windows machine and reinstalled using my 'seed' and waited while it did a full rescan. Now the Windows machine only has the 2 deposits to the same address as with the MacBook.

There are no other transactions so the coin has not been moved from the wallet.

In order to use my 'seed' to recover the wallet I had renamed the .dat file. I can retrieve the original deposits by putting the original .dat file back. But that does not help with the password issue.

I would certainly appreciate any help or suggestions.
Thank you.
